rpc/src/responses/block_ids_in_chain.rs

```diff
@@ -36,7 +38,10 @@ mod test {
         let mock_response = r#"["blockId1"]"#;
 
         let blocks_response = BlocksInChainResponse::from_response_str(mock_response);
-        assert!(blocks_response.is_err());
+        assert!(blocks_response.is_ok());
+
+        let blocks = blocks_response.unwrap().block_ids.into_vec();
+        assert!(blocks.len() == 1);
```

Comment:
I dont think the test name matches what this is doing anymore

Reply:
I totally missed this, good catch!